rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java

```diff
@@ -256,4 +256,21 @@ public static void addInputFromMatchingOutputScript(BtcTransaction transaction,
         searchForOutput(outputs, expectedOutputScript)
             .ifPresent(transaction::addInput);
     }
+
+    public static TransactionWitness addBaseWitnessToBtcTxInputThatSpendsFromRedeemScript(Script redeemScript) {
```

Comment:
```suggestion
    public static TransactionWitness createBaseWitnessThatSpendsFromRedeemScript(Script redeemScript) {
```

you are not actually adding the witness to the input, right?

Reply:
we could have two methods, this one and one also receiving the input index and the tx, wdyt?

Reply:
> you are not actually adding the witness to the input, right?

done! beforehand we added the witness, but to separate responsibilities I left this method so it only creates the witness